app/pandas.ipynb

```diff
@@ -458,36 +416,32 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
     "s = pd.Series([\"numpy\", \"pandas\", \"matplotlib\"])"
    ],
    "metadata": {
     "collapsed": false
-   }
+   },
+   "outputs": [],
+   "execution_count": null
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
-   "source": [
-    "# make all words in uppercase"
-   ],
+   "source": "s = s.str.upper()",
```

Comment:
Converting the Series to uppercase overwrites the original data. If you need to perform further operations, consider storing the result in a new variable.